docs/content/installation/install_driver_operatorhub.md

```diff
@@ -2,6 +2,10 @@
 
 When using OperatorHub.io, the operator for IBM® block storage CSI driver can be installed directly from the OperatorHub.io website. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
 
-To install the CSI driver from OperatorHub.io, go to https://operatorhub.io/operator/ibm-block-csi-operator-community and follow the installation instructions, once clicking the **Install** button.
+1. Install the CSI operator from OperatorHub.io, go to https://operatorhub.io/operator/ibm-block-csi-operator-community and follow the installation instructions, once clicking the **Install** button.
 
-**Note:** To ensure that the operator installs the driver, be sure to apply the YAML file that is located as part of the ibm-block-csi-operator-community page mentioned above.
+2. Apply the IBM block storage CSI driver custom resource definition yaml provided.
+
+    **Note:** To ensure that the operator installs the driver, be sure to apply the YAML file that is located as part of the ibm-block-csi-operator-community page mentioned above.
+
+3. (Optional) Apply the HostDefiner custom resource definition yaml provided. For more information, see [Configuring the host definer](../configuration/configuring_hostdefiner.md).
```

Comment:
same as for CR, we should either use `IBMBlockCSI` for the driver CRD, or "host definer" for the `HostDefiner` CRD, to remain consistent

Reply:
updated